### PR TITLE
Add Modrinth badge to README.md

### PR DESCRIPTION
**Pull request by Enderteck**
Here's the badge. I used [Shields.io](https://shields.io/) as there is modrinth support there.

Modrinth Downloads

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,9 +2,9 @@
 
 # Embeddium
 
+![Modrinth Downloads](https://img.shields.io/modrinth/dt/sk9rgfiA?label=Modrinth&labelColor=%232D2D2D)
 [![](http://cf.way2muchnoise.eu/short_embeddium_downloads.svg)](https://www.curseforge.com/minecraft/mc-mods/embeddium)
 [![](http://cf.way2muchnoise.eu/versions/Available%20for_embeddium_full.svg)](https://www.curseforge.com/minecraft/mc-mods/embeddium/files)
-
```
Comment (embeddedt) [inline README.md:7]: The blank line is important, otherwise this happens:
image

Comment (Enderteck) [reply README.md:7]: My bad didn't see that

**Comment by Enderteck** [comment]
Should be fixed and pr done

**Comment by embeddedt** [comment]
Clicking the link takes me to the image rather than the Modrinth page.

This is where I'm viewing the modified version: https://github.com/Enderteck/embeddium/tree/patch-1?tab=readme-ov-file

**Comment by Enderteck** [comment]
Weird behavior. I'll switch to a direct link or html instead of markdown. If you click the badge in the new tab, does it work ?

**Comment by Enderteck** [comment]
Fixed it by changing the link to be in the md file instead of on the image. Should work now:

https://github.com/Enderteck/embeddium/tree/patch-1?tab=readme-ov-file

**Pull request merged**